KeystoreAgent.js

```diff
@@ -196,6 +196,22 @@ export class KeystoreAgent {
     return new KeyAgreementKey(
       {id, kmsId, type, capability, invocationSigner, kmsClient});
   }
+
+  /**
+   * Update a keystore using the given configuration.
+   *
+   * @param {object} options - The options to use.
+   * @param {string} [options.capability=undefined] - The ZCAP authorization
```

Comment:
```suggestion
   * @param {string} [options.capability] - The ZCAP authorization
```
The `=undefined` part is implied whenever a param is optional.